### ISSUE TITLE
Pull request (PR) creation

### ISSUE THREAD

**Issue by avashchylau-io** [ljcmps01/zephyr-course#1]
Hi @ljcmps01! Just wanted to kindly remind that you are free to open the PR when you ready so I could leave the comments for the specific code lines.

Thanks!

**Comment by ljcmps01** [comment]
Yes sorry, I'm preparing for some finals at the uni so I'm following the course's classes and taking notes, as soon as I have the chance I'll do the homeworks.

Sorry for the delay.

**Comment by avashchylau-io** [comment]
> Yes sorry, I'm preparing for some finals at the uni so I'm following the course's classes and taking notes, as soon as I have the chance I'll do the homeworks.
> 
> Sorry for the delay.

No, problem. Just wanted to make sure you don't have any troubles with that. 
Good luck with your exams!
